discord-scripts/invite-management.ts

```diff
@@ -93,11 +56,8 @@ export default async function sendInvite(discordClient: Client, robot: Robot) {
         return
       }
 
-      const maxAge =
-        ((interaction.options.get("max-age", false)?.value as number) ?? 24) *
-        3600
-      const maxUses =
-        (interaction.options.get("max-uses", false)?.value as number) ?? 10
+      const maxAge = WEEK // default 7 days
```

Comment:
Ditto re `1 * WEEK`—though you can also do `7 * DAYS` or whatever. It's a little weird to have the code say 1 week and the comment say 7 days :)